### ISSUE TITLE
Using composer for installing other libraries

### ISSUE THREAD

**Issue by mipapo** [Runalyze/Runalyze#1283]
```
{
    "require": {
        "phpfastcache/phpfastcache": "3.0.3",
        "components/font-awesome": "4.3.*@dev"
   }
}
```
The autoload.php just has to be added to our autoloader .

**Comment by laufhannes** [comment]
I guess there is no possibility to change files of these dependencies without forking them and using our fork? (As we need the `file_exists` check for phpFastCache, see #1281)

**Comment by astax-t** [comment]
@laufhannes , it's better to just make a pull request for phpFastCache library so this is fixed in 3.0.4 or whatever next version is released.

BTW, this also means it's better to specify dependency as

      "phpfastcache/phpfastcache": "3.0.x",

or 

      "phpfastcache/phpfastcache": "^3.0.3",

**Comment by laufhannes** [comment]
Composer is already used for swiftmailer and email validator at least.
